recipes/yder/all/conanfile.py

```diff
@@ -30,27 +32,19 @@ class YderConan(ConanFile):
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
+        if self.settings.os != "Linux":
```

Comment:
Just to confirm, the fix comes from giving access to libsystemd for every other OS, rather than just Windows. Does this mean that things like FreeBDS would also have broken? (Or just never worked?)

Thanks!

Reply:
The libsystemd recipe is Linux only: https://github.com/conan-io/conan-center-index/blob/29d0e1d7af1ab735ddc1a8bfbfa274b21417ec5f/recipes/libsystemd/all/conanfile.py#L68-L70

So every other OS (except Windows which somehow deleted the option) trying to build with the default option value (True) would have failed. One could build for macOS/FreeBSD outside of CCI by specifying `with_libsystemd=False`.

Reply:
Yeah that it what was messing with me, it was weird having a non linux chekc there, thanks for the clarification :)